docroot/profiles/custom/cgov_site/modules/custom/cgov_cancer_research/config/install/core.entity_view_display.paragraph.cgov_external_link.cancer_research_list_item.yml

```diff
@@ -4,6 +4,7 @@ dependencies:
   config:
     - core.entity_view_mode.paragraph.cancer_research_list_item
     - field.field.paragraph.cgov_external_link.field_external_link
+    - field.field.paragraph.cgov_external_link.field_external_link_display
```

Comment:
@bryanpizzillo this file changed and I'm not sure it was supposed to.

Reply:
THIS IS PROBABLY OK, AND THIS COMMENT CAN BE RESOLVED.

This file should not be changed, but also, it would change if it was not actually updated when field_external_link_display got added in #4647.